src/app/ecosystem-explorer/project-form/components/EcosystemProjectForm.tsx

```diff
@@ -119,7 +119,7 @@ export function EcosystemProjectForm({
           placeholder="Select year"
           options={yearOptions}
           disabled={isSubmitting}
-          innerWidth="w-40"
+          buttonWidth="w-40"
```

Comment:
Let's see if we can just have it be as wide as the button text.

Reply:
It causes the button to collapse when selecting an option, which is not great. So, I think we should stick with the current implementation for now.

https://share.cleanshot.com/YFBs6kYx

Reply:
Added to the backlog!